lib/rules/new-line-between-multi-line-property.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * @fileoverview Enforce new lines between multi-line properties in Vue components.
+ * @author IWANABETHATGUY
+ */
+'use strict'
+
+const utils = require('../utils')
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+// @ts-ignore
+module.exports = {
+  meta: {
+    type: 'layout',
+    docs: {
+      description:
+        'enforce new lines between multi-line properties in Vue components',
+      categories: undefined,
+      url:
+        'https://eslint.vuejs.org/rules/new-line-between-multi-line-property.html'
+    },
+    fixable: 'whitespace', // or "code" or "whitespace"
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          // number of line you want to insert after multi-line property
+          minLineOfMultilineProperty: {
+            type: 'number',
+            minimum: 2
+          }
+        }
+      }
+    ]
+  },
+
+  /** @param {RuleContext} context */
+  create(context) {
+    // always insert one line
+    const insertLine = 1
+
+    let minLineOfMultilineProperty = 2
+    if (
+      context.options &&
+      context.options[0] &&
+      context.options[0].minLineOfMultilineProperty
+    ) {
+      minLineOfMultilineProperty = context.options[0].minLineOfMultilineProperty
+    }
+
+    /** @type {any[]} */
+    const callStack = []
+    /** @type {any[]} */
+    let comments = []
+    return Object.assign(
+      {
+        /**
+         * @param {Program} node
+         */
+        Program(node) {
+          comments = node.comments
```

Comment:
You can use `context.getSourceCode().getAllComments()`.

https://eslint.org/docs/developer-guide/working-with-rules#contextgetsourcecode-1